Rocket.Chat/Managers/Model/MessageManager/MessageManager.swift

```diff
@@ -21,44 +21,6 @@ extension MessageManager {
 
     static var blockedUsersList = UserDefaults.group.value(forKey: kBlockedUsersIndentifiers) as? [String] ?? []
 
-    static func getHistory(_ subscription: UnmanagedSubscription, lastMessageDate: Date?, completion: @escaping MessagesHistoryCompletion) {
-        var lastDate: Any!
-
-        if let lastMessageDate = lastMessageDate {
-            lastDate = ["$date": lastMessageDate.timeIntervalSince1970 * 1000]
-        } else {
-            lastDate = NSNull()
-        }
-
-        let request = [
-            "msg": "method",
-            "method": "loadHistory",
-            "params": ["\(subscription.rid)", lastDate, historySize]
```

Comment:
@cardoso I think we can remove the reference of `historySize` from the file now.

Reply:
Nice catch